figma-shop/src/app/app-routing.module.ts

```diff
@@ -1,8 +1,22 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 
+import { ChatComponent } from './pages/chat/chat.component';
+import { BlogComponent } from './pages/blog/blog.component';
+import { AboutComponent } from './pages/about/about.component';
+import { CareersComponent } from './pages/careers/careers.component';
+import { ProductsComponent } from './pages/products/products.component';
 
-const routes: Routes = [];
+
+const routes: Routes = [
+  { path: '', redirectTo: 'products', pathMatch: 'full' },
+  { path: 'chat', component: ChatComponent },
+  { path: 'blog', component: BlogComponent },
+  { path: 'about', component: AboutComponent },
+  { path: 'careers', component: CareersComponent },
```

Comment:
You have duplicate `careers` routes. Remove one.

Reply:
Fix it, thank you